test/built-ins/Iterator/prototype/Symbol.toStringTag/prop-desc.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2023 Michael Ficarra. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 25.1.2.1
+description: Property descriptor
+info: |
+    ES6 Section 17
+
+    Every other data property described in clauses 18 through 26 and in Annex
+    B.2 has the attributes { [[Writable]]: true, [[Enumerable]]: false,
+    [[Configurable]]: true } unless otherwise specified.
+features: [Symbol.iterator]
```

Comment:
This is missing `iterator-helpers`, because it's the proposal that adds `Iterator.prototype` and `Symbol.toStringTag` on %IteratorPrototype%.